### ISSUE TITLE
Selecting of text in readonly fields not working when selecting text from right to left

### ISSUE THREAD

**Issue by metasnw** [metasfresh/metasfresh-webui-frontend#1797]
### Is this a bug or feature request?
bug

### What is the current behavior?
1. OK: if you double click on a value in a readonly field its fine
1. OK: if you select the text from left to right using mouse its also fine
1. NOK: if you select the text from right to right using mouse its not copying anything to clipboard

#### Which are the steps to reproduce?
1. have a partner record and set it to not active
1. select the companyname from right to left 
1. CTRL+C 
1. CTRL+V in any app
1. first time: works
1. do it again
1. NOK: not working anymore
1. work around: use double click or select the text from left to right

http://recordit.co/SkAjrmeLJp

### What is the expected or desired behavior?
should always work

**Comment by siemiatj** [comment]
@PabloSichert I did some debugging hoping it's a browser bug, but it isn't. As far as I remember the copy event is just not called (even though the document.selection is correct) so maybe for some weird reason something is silently preventing it. Maybe worth starting with a basic (not wrapped in our magic) disabled input field, and working down from there.

**Comment by PabloSichert** [comment]
I disabled JavaScript, copied the HTML markup via DevTools and removed all classes from the input. I was still able to reproduce the bug:

![select](https://user-images.githubusercontent.com/4450694/40801154-86947e1e-6512-11e8-8fb1-2da0c746f275.gif)

I'm afraid the fix for that is out of our scope.

I suggest closing this issue as `wontfix`

**Comment by siemiatj** [comment]
I would agree if not the fact that I've tested this in a sandbox and it's working fine, ie https://codesandbox.io/s/xrwqm5xyjo . But tbh I'm out of ideas and this seems to be inconsistent between the browsers, as Firefox won't allow copying from disabled fields at all.

**Comment by ionut-movila** [comment]
@metasnw I just tested this in latest Chrome 74 and it works ok. Also tested with latest Firefox 66 and there as Kuba mentioned disabled inputs won't let you copy the content. This can work in FF if we use readonly instead of disabled, as explained here: https://stackoverflow.com/a/22438511/8648758

Can you please re-test and close the issue if all good?
